docs/CONTRIBUTING.md

```diff
@@ -84,11 +84,21 @@ It is recommended to set it in your IDE auto formatting settings for this projec
 ````
 
 ### Build 
- After every successful build on travis, a Docker image is built. Every image is tagged twice:
- - `$TRAVIS_COMMIT` [travis' environment variable](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables)
- - `$TRAVIS_BRANCH` [travis' environment variable](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables)
- 
-This images are pushed to the docker hub repo: [commercetools/commercetools-project-sync](https://hub.docker.com/r/commercetools/commercetools-project-sync/)
+
+ Gradle docker plugin is used to build and deploy the docker images. 
+ To build the docker image locally, please run
+ ````bash
+./gradlew dockerBuildImage
+````
+The docker image has been built and published to your desktop docker.
+
+Example:
+For testing all resources, please run:
+````bash
+docker run commercetools/commercetools-project-sync:<version> -s all
+````
+
+For more detailed information on the build and release process, see [Build and Release](BUILD.md) documentation.
 
 ## License
-Copyright (c) 2019 commercetools
+Copyright (c) 2021 commercetools
```

Comment:
not sure but copyright statement year might be better if you don't touch it? 🤔 
see: https://stackoverflow.com/a/2391555

Reply:
Yes, I think it is better if we don't touch copyright statement year.
@ahalberkamp do you agree?

Reply:
yes leave it untouched

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/231/commits/bba622f7259e3ca338c0e8612c304f37bc32bdc2